docs/_generate/index.html

```diff
@@ -488,6 +489,67 @@ <h3 class='space-bottom1'><a href='#sources-video' title='link to video'>video</
             </tbody>
           </table>
         </div>
+        <div id='sources-canvas' class='pad2 keyline-bottom'>
+          <h3 class='space-bottom1'><a href='#sources-canvas' title='link to canvas'>canvas</a></h3>
+          <p>
+            A canvas source. The <code>"canvas"</code> value is the ID of the canvas element in the document.
```

Comment:
Hmm…much of this documentation _does_ contain `<code>"something"</code>` — it seems to generally apply to many JSON values https://www.mapbox.com/mapbox-gl-style-spec/#sources. I copied this style from video sources, right above it ([e.g.](https://github.com/mapbox/mapbox-gl-style-spec/blob/mb-pages/docs/_generate/index.html#L439-L444))